lua/gatekeeper/policylib.lua

```diff
@@ -299,3 +299,19 @@ function lpm_para_estimate(ipv4_file)
 
 	return num_rules, num_tbl8s
 end
+
+-- Estimate the number of rules and number of tbl8s in DPDK LPM6 library.
+function lpm6_para_estimate(ipv6_file)
```

Comment:
This function must go to `lua/examples/policy.lua` because it depends on the policy; more specifically, on the format of the prefix file, or from wherever the prefixes are obtained.